core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -69,6 +69,7 @@ $(call force,CFG_DT,y)
 CFG_SE_API ?= y
 CFG_SE_API_SELF_TEST ?= y
 CFG_PCSC_PASSTHRU_READER_DRV ?= y
+CFG_AES_GCM_TABLE_BASED ?= y
```

Comment:
Why only for QEMU? Shouldn't that be always enabled in `core/crypto.mk`  as long as `CFG_CRYPTO_WITH_CE` is not `y`?

Also it would be good to have performance numbers in the commit log. FWIW, here is the improvement I see on HiKey960 (with `CFG_CRYPTO_WITH_CE=n` of course, not a very realistic scenario since CE is much faster but for the sake of comparison it works):
```
root@HiKey960:/ xtest --aes-perf -m GCM
(CFG_AES_GCM_TABLE_BASED=n)
min=69.27us max=86.458us mean=70.5695us stddev=0.955826us (cv 1.35445%) (13.8383MiB/s)
(CFG_AES_GCM_TABLE_BASED=y)
min=41.666us max=53.646us mean=42.138us stddev=0.621345us (cv 1.47455%) (23.1753MiB/s)
```

Reply:
Thanks, I'll update accordingly (using your numbers)